src/main/resources/application.properties

```diff
@@ -1 +1,4 @@
-
+spring.jpa.hibernate.ddl-auto=create
+spring.datasource.url=jdbc:mysql://localhost:3306/keyboard_builder
```

Comment:
This assumes you have WAMP running with a database setup called "keyboard_builder"